### DISCUSSION TITLE
File upload looks

### DISCUSSION THREAD

**Discussion by JimTudeski** [primefaces/community#1346] [PrimeNG]
After i upgrade from 12 to 17, all file upload components looks like:
![image](https://github.com/primefaces/community/assets/11393403/d95b14c4-cf94-4f87-936a-1101f738f378)
don't know why...

**Comment by JimTudeski** [comment]
I checked, version 15 is the last when the button is ok, but it's wrong with 16 and 17.

**Comment by JimTudeski** [comment]
input[type=file] {
  display: none !important;
}

solved temporarily in the styles.scss but not so beautiful

**Comment by cetincakiroglu** [comment]
Hi,

Could you please share a stackblitz example so we can identify the issue? I couldn't replicate it with the latest (17.10.0)

**Comment by cetincakiroglu** [comment]
Closing the discussion due to inactivity.
